automol/util/_util.py

```diff
@@ -189,34 +190,48 @@ def move_items_to_front(lst: Sequence, items) -> tuple:
     return tuple(lst)
 
 
-def breakby(lst: Collection, elem) -> tuple[tuple, tuple]:
+def breakby(lst: Sequence, elem) -> tuple[tuple, tuple]:
```

Comment:
Parameter types look good. The return should be
```
-> tuple[tuple, ...]
```
The ellipsis indicates that it could be any number of tuples.